stl/inc/format

```diff
@@ -2074,6 +2074,145 @@ wstring format(const locale& _Loc, wstring_view _Fmt, const _Types&... _Args) {
     return _STD vformat(_Loc, _Fmt, _STD make_wformat_args(_Args...));
 }
 
+template <class _OutputIt, class _CharT>
+struct _Format_to_n_iterator {
+    _OutputIt _Out;
+    iter_difference_t<_OutputIt> _Max;
+    iter_difference_t<_OutputIt> _Count = 0;
+
+    using iterator_category = output_iterator_tag;
+    using value_type        = void;

```

Comment:
After re-reading the requirements, I think all we need to provide is a `difference_type`, so that we satisfy the `weakly_incrementable` concept, which is need for `ouptut_iterator` (a constraint on `vformat_to`, amongst others).

Reply:
No we don't / Yes, I believe that's correct.